ocaml/xe-cli/bash-completion

```diff
@@ -280,15 +277,12 @@ _xe()
 
                 if [[ "${fst}" == "vbd" ]]; then
                     set_completions 'Disk,CD' "$value"
-                    return 0
                 elif [[ "${fst}" == "vdi" ]]; then
                     set_completions 'system,user,suspend,crashdump' "$value"
-                    return 0
                 elif [[ "${fst}" == "sr" ]]; then
-                    local words=$( ${xe} sm-list params=type --minimal 2>/dev/null )
-                    set_completions "$words" "$value"
-                    return 0
+                    set_completions_for_names "sm-list" "type" "$value"
```

Comment:
This change doesn't seem related to the cleanup either

Reply:
it's just cleaning up duplicate code. the function does these commands already.